rhel6/guide/services/ssh/sshd_approved_macs.var

```diff
@@ -0,0 +1,14 @@
+documentation_complete: true
+
+title: 'SSH Approved MACs by FIPS'
+
+description: "Specify the FIPS approved MACs (message authentication code) algorithms\n\tthat are used for data integrity protection by the SSH server."
```

Comment:
Why is the \t\n here?

Reply:
Some of the descriptions use \n and \t and some are indented and use the `|-` yaml syntax. At some point we should reflow everything using `|-` however that's not in the scope of this PR. This PR just puts RHEL6 guide closer to Linux OS so it's using the Linux OS's guide as is.